packages/gatsby-plugin-preact/src/gatsby-node.js

```diff
@@ -1,5 +1,20 @@
 const PreactRefreshPlugin = require(`@prefresh/webpack`)
 
+const FRAMEWORK_BUNDLES_PREACT = [
+  `preact`,
```

Comment:
Main change: Adding `preact` to this regex and then just using this regex instead of trying to reuse what Gatsby framework itself uses